mitiq/pt/tests/test_pt.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (C) Unitary Fund
+#
+# This source code is licensed under the GPL license (v3) found in the
+# LICENSE file in the root directory of this source tree.
+
+import random
+
+import cirq
+import qiskit
+import numpy as np
+import networkx as nx
+
+from mitiq.pt.pt import (
+    twirl_CNOT_gates,
+    twirl_CZ_gates,
+    execute_with_pauli_twirling,
+    CNOT_twirling_gates,
+    CZ_twirling_gates,
+)
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+from mitiq.benchmarks import generate_mirror_circuit
+
+num_qubits = 2
+qubits = cirq.LineQubit.range(num_qubits)
+circuit = cirq.Circuit()
+circuit.append(cirq.CNOT.on_each(zip(qubits, qubits[1:])))
+
```

Comment:
This should fix coverage errors for CZ functions:

```suggestion
circuit.append(cirq.CNOT.on_each(zip(qubits, qubits[1:])))
circuit.append(cirq.CZ.on_each(zip(qubits, qubits[1:])))
```
or,  maybe equivalently:

```python
circuit.append(cirq.CNOT(*qubits))
circuit.append(cirq.CZ(*qubits))
```

Reply:
Niiiice! Looks like that's the reason why coverage was complaining.

I originally wrote line 26 when I had `len(qubits) > 2` so calling `cirq.CNOT(*qubits)` does not work, but since it's just 2 I like your simplified approach. I'm not sure I even knew you could call the gates directly. I thought you had to use `cirq.GATE.on`. Good to know.